pyanaconda/ntp.py

```diff
@@ -106,7 +110,12 @@ def ntp_server_working(server_hostname):
     client = ntplib.NTPClient()
 
     try:
-        client.request(server_hostname)
+        # ntplib doesn't support NTS
+        if "nts" in options:
+            s = socket.create_connection((server_hostname, 4460), 2)
```

Comment:
How can I test it? I wasn't able to find a server that would be marked as working.

Reply:
You can test it with `nts-test.strangled.net` and `ntp1.glypnod.com` for example. Cloudflare and others will hopefully move to the new port soon. Please note that we will also need to update the chrony package to use this port by default.